.github/workflows/comments.yml

```diff
@@ -119,6 +121,39 @@ jobs:
               if (!$commentToPost -and !$labelToAdd) {
                 Write-Host "Neither 'commentToPost' nor 'labelToAdd' was set, so not notifying anybody"
               }
+
+              # this is where check if a comment id is set, if so call minimize comment here
+              if ($commentId) {
+                Write-Host "Minimizing comment"
+
+                $mutation =@"
+                      mutation {
+                          __typename
+                          minimizeComment(
+                              input: {
+                                  subjectId: "$($commentId)",
+                                  classifier: ABUSE
+                              }
+                          ) {
+                              clientMutationId
+                          }
+                      }
+                "@
+
+                $payload = @{
+                    query=$mutation
+                }
+
+                $headers = New-Object "System.Collections.Generic.Dictionary[[String],[String]]"
+                $headers.Add("content-type","application/json")
+                $headers.Add("Authorization","bearer $Env:GITHUB_TOKEN")
+
+                $body = ConvertTo-Json $payload
+                $url = "https://api.github.com/graphql"
+                $response= Invoke-RestMethod -Uri $url -Headers $headers -Method "POST" -Body $body
+
+                Write-Host "Minimized comment"
```

Comment:
Can share a link to a place you tried it?

Reply:
the feature works now! Made the feature opt in, similar to how we handle commentToPost/labelToAdd parameters.

> Can share a link to a place you tried it?

tested it using a sample repo: https://github.com/haritha-mohan/heat-sensor-tester/pull/1

the reason why it just shows "minimized" as opposed to "marked as abuse" is due to how GraphQL processes the action internally: see ref https://github.com/orgs/community/discussions/19865

also in case anyone uses the REST API/graph QL in the future, there is a distinction between the id vs node id🙃: https://docs.github.com/en/graphql/guides/using-global-node-ids